src/build.py

```diff
@@ -1103,6 +1103,7 @@ def TestBare():
       runner=os.path.join(INSTALL_BIN, 'wasm.opt'),
       indir=s2wasm_out,
       fails=SPEC_KNOWN_TORTURE_FAILURES,
+      warn_only=True,  # TODO all the spec tests fail with syntax error
```

Comment:
It still fails with the 0xc branch, but at least it's a more accurate error message?

So here I'm thinking, pin to 0xc, leave `warn_only=True`, link the spec bug in the TODO